pgt/src/PGT/Output/examples/query-plan.sql.expected

```diff
@@ -0,0 +1,69 @@
+       Column        |                Type                | Collation | Nullable | Default
+---------------------+------------------------------------+-----------+----------+---------
+ id                  | uuid_not_null                      |           |          |
+ created_at          | timestamptz_not_null               |           |          |
+
+-- Example comment 1
+-- (1 row)
+
+-[ RECORD 1 ]---------------+------------------------------------------------------------
+id               | 00000000-0000-7000-0000-000000000137
+created_at       | 1970-01-02 02:02:10
+
+-- Example comment 2
+-- another sub-comment
+-- (2 row)
+
+-[ RECORD 1 ]---------------+------------------------------------------------------------
+id               | 00000000-0000-7000-0000-000000000137
+created_at       | 1970-01-02 02:02:10
+-[ RECORD 2 ]---------------+------------------------------------------------------------
+id               | 00000000-0000-7000-0000-000000000138
+created_at       | 1970-01-02 02:02:11
+
+             QUERY PLAN
+-------------------------------------
+ [                                  +
+   {                                +
+     "Plan": {                      +
+       "Node Type": "Result",       +
+       "Parallel Aware": false,     +
+       "Startup Cost": 0.00,        +
+       "Total Cost": 0.01,          +
+       "Plan Rows": 1,              +
+       "Plan Width": 4,             +
+       "Actual Startup Time": 0.001,+
+       "Actual Total Time": 0.001,  +
+       "Actual Rows": 1,            +
+       "Actual Loops": 1,           +
+       "Output": ["1"],             +
+       "Shared Hit Blocks": 0,      +
+       "Shared Read Blocks": 0,     +
+       "Shared Dirtied Blocks": 0,  +
+       "Shared Written Blocks": 0,  +
+       "Local Hit Blocks": 0,       +
+       "Local Read Blocks": 0,      +
+       "Local Dirtied Blocks": 0,   +
+       "Local Written Blocks": 0,   +
+       "Temp Read Blocks": 0,       +
+       "Temp Written Blocks": 0     +
+     },                             +
+     "Planning": {                  +
+       "Shared Hit Blocks": 0,      +
+       "Shared Read Blocks": 0,     +
+       "Shared Dirtied Blocks": 0,  +
+       "Shared Written Blocks": 0,  +
+       "Local Hit Blocks": 0,       +
+       "Local Read Blocks": 0,      +
+       "Local Dirtied Blocks": 0,   +
+       "Local Written Blocks": 0,   +
+       "Temp Read Blocks": 0,       +
+       "Temp Written Blocks": 0     +
+     },                             +
+     "Planning Time": 0.028,        +
```

Comment:
Instance of: https://github.com/mbj/mhs/pull/189/files#r1169029086

Reply:
@mbj The idea is to attempt to normalize some of them by rounding them to the nearest tenth to quantities I am currently calling magnitudes. Disclaimer, I haven't tested this out on a large scale. If it doesn't work out we can filter them out. 

See https://github.com/mbj/mhs/pull/189/files#diff-1e7b83122d11c592d5f904747b83733c1f28d0615708beadcf86cacf7c7ad892R27